File: backend-orchestrator/src/helpers/get-content-type-for-screenshot-format.ts
import { ScreenshotFormat } from "../schemas/screenshot-format-schema.js";

export const getContentTypeForScreenshotFormat = (
  format: ScreenshotFormat
): string => {
  if (format == ScreenshotFormat.bmp_raw) {
    return "application/octet-stream";
  }
  return `image/${format}`;
};

File: screenshot-processer/src/helpers/get_content_type_for_screenshot_format.py
from src.enums.screenshot_format import ScreenshotFormat


def get_content_type_for_screenshot_format(format: ScreenshotFormat) -> str:
    if format == ScreenshotFormat.bmp_raw:
        return "application/octet-stream"
    return f"image/{format.value}"
